build.assets/Makefile

```diff
@@ -325,18 +383,33 @@ release-windows-unsigned: buildbox
 #
 # Run docs tester to detect problems.
 #
-.PHONY: docsbox
+.PHONY:docsbox
 docsbox:
 	if ! docker inspect --type=image $(DOCSBOX) 2>&1 >/dev/null; then docker pull $(DOCSBOX) || true; fi
 
-.PHONY: test-docs
-test-docs: DOCS_VERSION := $(shell grep -E ^VERSION $(MAKEFILE_ROOT_DIR)/Makefile | cut -d= -f2 | cut -d. -f1-2)
```

Comment:
This has always failed and emitted a warning: because `MAKEFILE_ROOT_DIR` was never defined:

`grep: /Makefile: No such file or directory`

As a result, the container would just mount `src/content` and run docs checks against all versions.

With this change, we get the same behavior that we've always had, without the pesky grep warning.